elements/rh-tile/demo/accented-tiles.html

```diff
@@ -0,0 +1,66 @@
+<rh-context-demo>
+  <rh-tile-group class="accented-tiles">
+    <rh-tile>
+      <h2 slot="headline"><a href="#top">Link</a></h2>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </rh-tile>
+
+    <rh-tile>
+      <h2 slot="headline"><a href="#top">Link 2</a></h2>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </rh-tile>
+
+    <rh-tile>
+      <h2 slot="headline"><a href="#top">Link 2</a></h2>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    </rh-tile>
+  </rh-tile-group>
+</rh-context-demo>
+
+<script type="module">
+  import '@rhds/elements/rh-tile/rh-tile.js';
+  import '@rhds/elements/lib/elements/rh-context-demo/rh-context-demo.js';
+</script>
+
+<link rel="stylesheet" href="../rh-tile-lightdom.css">
+
+<style>
+  rh-context-demo::part(demo) {
+    display: flex;
+    align-items: top;
+    flex-wrap: wrap;
+    gap: var(--rh-space-2xl, 32px);
+  }
+
+  rh-tile-group {
+    display: flex;
+    align-items: top;
+    flex-wrap: wrap;
+    column-gap: var(--rh-space-2xl, 32px);
+    row-gap: var(--rh-space-2xl, 32px);
+    margin-inline-end: var(--rh-space-md, 8px);
+    & > * {
+      width: 320px;
+    }
+  }
+
+  .accented-tiles {
+    & rh-tile {
+      position: relative;
+      overflow: hidden;
+      border-radius: var(--rh-border-radius-default, 3px);
+      &::before {
+        content: '';
+        position: absolute;
+        z-index: 2;
+        display: block;
+        inset-block-start: 0;
+        inset-inline: 0;
+        border-block-start-color: var(--rh-color-brand-red-on-light, #ee0000);
+        border-block-start-width: 3px;
+        border-block-start-style: solid;
+        pointer-events: none;
+      }
+    }
+  }
```

Comment:
IIRC we are still on the stance of not using CSS nesting in production code.  I suggest rewriting this in plain CSS format for now. The holdout was ESR Firefox being 115 which didn't support CSS Nesting.  Given the recent change of Firefox ESR becoming v128, we should be able to do all CSS in nested format soon.

Reply:
Good point. I just took Benny's code mostly verbatim.

Looking at this again, I think what I've got in mind is probably a better approach or at least one more consistent to how implementors might apply this anyway.